usage/bigquery.go

```diff
@@ -166,6 +208,29 @@ func buildUsageSummaryQuery(table string, userID string, creatorID string, spec
 	return sql, args, nil
 }
 
+func buildTotalUsageSummaryQuery(table string, spec QuerySpec) (string, []interface{}, error) {
+
+	query := squirrel.Select(
+		"date_ts,date_s,week_ts,week_s,volume_eth,volume_usd, fee_derived_minutes,participation_rate,inflation,active_transcoder_count,delegators_count,average_price_per_pixel,average_pixel_per_minute").
```

Comment:
Are these _all_ the fields? Why not use `*`?

Reply:
Confirmed that they are not all the fields